src/features/dashboard/usage/cost-card.tsx

```diff
@@ -22,15 +22,19 @@ export function CostCard({
       <CardHeader>
         <CardTitle className="font-mono">Usage Costs</CardTitle>
         <CardDescription>
-          Total cost of all resources for the current billing period.
+          Total cost of all resources for the current billing month.
```

Comment:
current billing month sounds wrong, maybe "Total cost of all resources this month."